### PR TITLE
Bugfix/json array request body fix

### PR DESCRIPTION
**Pull request by dsgr**
https://github.com/adamyork/wiremock-velocity-transformer/issues/14

### CODE REVIEW

File: src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ private Context addBodyToContext(final String body, final Context context) {
                 .orElseGet(() -> {
                     final JSONParser parser = new JSONParser(JSONParser.MODE_JSON_SIMPLE);
                     return Unchecked.supplier(() -> {
-                        final JSONObject json = (JSONObject) parser.parse(body);
+                        final Object json = parser.parse(body);
```
Comment (adamyork) [inline src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java:137]: I don't think down casting to the most primitive type in java is what we want to do here. You are basically disregarding type safety. There are really only two valid wrapper types per the JSON rfc; [] (JSONArray) and {} (JSONObject). There may be an interface we can assign the cast resulted of parse to instead.

**Review by adamyork — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java (hunk 1 above)
Comment (dsgr) [reply src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java:137]: But parse method declared as 
`public Object parse(String in) throws ParseException`
and method\`s return type is Object. So I offer don\`t cast to any other type.
Also I had test this variant and all works fine.

Comment (adamyork) [reply src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java:137]: okay i see. the library its using needs to be swapped out with something else. I assumed this was using gson. I think gson provides a better api here , https://static.javadoc.io/com.google.code.gson/gson/2.6.2/com/google/gson/JsonParser.html

Comment (adamyork) [reply src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java:137]: hi , please try the latest release : https://github.com/adamyork/wiremock-velocity-transformer/releases/tag/2.3-release , and let me know if this fixes your issue.

Comment (dsgr) [reply src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java:137]: all works fine, thanks!

**Pull request closed without merging**